ui/src/app/ui-banner/ui-banner.scss

```diff
@@ -0,0 +1,53 @@
+.ui_banner {
+    line-height: 50px;
+    background: linear-gradient(to bottom, #333, #444, #555, #666, #999);
+    box-shadow: inset -2px 0 4px 0 rgba(0, 0, 0, 0.59);
+    transition: right .5s;
+    position: fixed;
+    width: 100%;
+    z-index: 12;
+    align-items: center;
+    opacity: 0.9;
+    text-align: center;
+    color: #ffffff;
```

Comment:
Use `color: white` here instead to make it more readable?

Reply:
Thanks @keithchong  will make the necessary changes.